security/security-design/fn-datasafe-dbaudit-to-oci-logging/terraform/function/oci-datasafe-audit-to-logging/func.py

```diff
@@ -0,0 +1,429 @@
+###############################################################################
+# Copyright (c) 2022, 2023, Oracle and/or its affiliates.  All rights reserved.
+# This software is dual-licensed to you under the Universal Permissive License
+# (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl.
+###############################################################################
+#
+# Author: Fabrizio Zarri
+#
+################################################################################
+
+import json
+import oci
+import io
+import logging
+import pandas as pd
+
+from oci.util import to_dict
+from fdk import response
+from datetime import datetime, timedelta
+
+#Set Logging level.
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+#logger.setLevel(logging.DEBUG)
+
+
+def init_rps_client():
+    try:
+        rps_client = oci.auth.signers.get_resource_principals_signer()
+    except Exception as e:
+        logger.error("Init rps client Error: %s", e)
+        raise
+    else:
+        logger.debug("rps client initialized")
```

Comment:
English sentences start with a capital letter and ends with a period ('.').

Reply:
I have corrected all comments and sentences.